x-pack/plugins/security_solution/public/cases/components/case_action_bar/index.tsx

```diff
@@ -101,17 +101,17 @@ const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
         <EuiDescriptionList compressed>
           <EuiFlexGroup gutterSize="l" alignItems="center">
             <EuiFlexItem>
+              <EuiDescriptionListTitle>
```

Comment:
This clears up the following error that showed in `cases/components/case_view/index.test.tsx`

```
Warning: validateDOMNesting(...): <dt> cannot appear as a descendant of <dd>.
  in dt (created by EuiDescriptionListTitle)
  in EuiDescriptionListTitle (created by CaseActionBarComponent)
  in dd (created by EuiDescriptionListDescription)
  in EuiDescriptionListDescription (created by CaseActionBarComponent)
  in div (created by EuiFlexItem)
```